streaming/java/streaming-runtime/src/main/java/io/ray/streaming/runtime/config/master/SchedulerConfig.java

```diff
@@ -0,0 +1,33 @@
+package io.ray.streaming.runtime.config.master;
+
+import io.ray.streaming.runtime.config.Config;
+
+/**
+ * Configuration for job scheduler.
+ */
+public interface SchedulerConfig extends Config {
+
+  String WORKER_INITIATION_WAIT_TIMEOUT_MS = "streaming.scheduler.worker.initiation.timeout.ms";
+  String WORKER_STARTING_WAIT_TIMEOUT_MS = "streaming.scheduler.worker.starting.timeout.ms";
+
+  /**
+   * The timeout ms of worker initiation.
+   * Default is: 10000ms(10s).
+   *
+   * @return timeout ms
+   */
+  @Key(WORKER_INITIATION_WAIT_TIMEOUT_MS)
+  @DefaultValue(value = "10000")
+  int workerInitiationWaitTimeoutMs();
+
+  /**
+   * The timeout ms of worker starting.
+   * Default is: 10000ms(10s).
+   *
+   * @return timeout ms
+   */
+  @Key(WORKER_STARTING_WAIT_TIMEOUT_MS)
+  @DefaultValue(value = "10000")
+  int workerStartingWaitTimeoutMs();
```

Comment:
What is different between ```Initiation``` and ```Starting```? It is better to rename ```Creating``` to  ```Starting```?

Reply:
Can not use ray.wait() on actor creating.

Initiation -> Use wait() to update worker's context.
Starting -> Use wait() to start worker's task.